dataprofiler/profilers/float_column_profile.py

```diff
@@ -105,23 +105,39 @@ def _get_float_precision(cls, df_series):
         :return: string representing its precision print format
         :rtype: int
         """
-        integer_decimal_loc = -1
-        float_precision = None
+        if df_series.empty:
```

Comment:
If it's empty, should it be None?

Reply:
Currently, our default value is 0. Personally, I think `None` is best for default as well if we want to change.

Reply:
My concern is this may actually error as well:

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.empty.html

```python
>>> df_empty = pd.DataFrame({'A' : []})

>>> df_empty
Empty DataFrame
Columns: [A]
Index: []

>>> df_empty.empty
True
```

Reply:
```python
>>> df = pd.DataFrame({'A' : [np.nan]})

>>> df
    A
0 NaN

>>> df.empty
False

>>> df.dropna().empty
True
```

Reply:
Plz switch off `.empty`